.changeset/poor-pianos-marry.md

```diff
@@ -0,0 +1,16 @@
+---
+'xstate': patch
+'@xstate/analytics': patch
+'@xstate/fsm': patch
+'@xstate/graph': patch
+'@xstate/immer': patch
+'@xstate/inspect': patch
+'@xstate/react': patch
+'@xstate/scxml': patch
+'@xstate/test': patch
+'@xstate/viz-example': patch
+'@xstate/viz': patch
+'@xstate/vue': patch
+---
+
+TypeScript version updated to 4.1
```

Comment:
This changeset should rather be removed - the TS upgrade in our repo has no effect on our consumers (just yet).